scripts/publish.sh

```diff
@@ -29,12 +29,12 @@ npm install
 
 # Node client
 bump_and_npm_publish "@nangohq/node" "$VERSION"
-npm update @nangohq/node -w @nangohq/shared
+npm install @nangohq/node@latest -w @nangohq/shared
 
 # Shared
 node scripts/flows.js
 bump_and_npm_publish "@nangohq/shared" "$VERSION"
-npm update @nangohq/shared -w nango -w @nangohq/nango-server -w @nangohq/nango-jobs @nangohq/nango-runner
+npm install @nangohq/shared@latest -w nango -w @nangohq/nango-server -w @nangohq/nango-jobs @nangohq/nango-runner
```

Comment:
shouldn't the cli also get the updated shared package?

Reply:
`-w nango` that's the CLI